yt/data_objects/construction_data_containers.py

```diff
@@ -495,6 +534,7 @@ def __init__(
         method="integrate",
         field_parameters=None,
         max_level=None,
+        moment=1,
         *,
```

Comment:
```suggestion
        *,
        moment=1,
```

Reply:
We need a quick explanation somewhere of why the use of the asterisk is so important and the rationale for putting kwargs before and after it is. Most developers are not fully familiar with it yet (I am myself not) and we need clarification on whether this is a style choice or a choice that the project needs to be implementing across the board.

Reply:
I realize this is not exactly a "new" python feature either--it's just not one that we've really required yt developers to use.

Reply:
It may not be new, but it *does* require Python 3, so it wasn't something we *could* use in yt until early 2020 when we dropped Python 2, which is why it's currently very sporadically used.
Anyway I'd happily open a discussion or explain my general observations/advice about this, where would be a good place for this ?

Reply:
I'd just open a github issue

Reply:
here you go #3992